prisma/schema.prisma

```diff
@@ -92,6 +101,52 @@ model Event {
   updatedAt DateTime @updatedAt
 }
 
+model Donor {
+  id                 Int            @id @default(autoincrement())
+  userId             Int?
+  email              String?
+  identification     String?
```

Comment:
What are these identifications?

Reply:
La cedula jajaj I didn't put it as the actual DB id just for consistency